XamlControlsGallery/Helper/ProtocolActivationClipboardHelper.cs

```diff
@@ -31,13 +31,13 @@ public static bool ShowCopyLinkTeachingTip
 
         public static void Copy(ControlInfoDataItem item)
         {
-            var uri = new Uri($"xamlcontrolsgallery://item/{item.UniqueId}", UriKind.Absolute);
+            var uri = new Uri($"winui2gallery://item/{item.UniqueId}", UriKind.Absolute);
```

Comment:
winuigallery

Reply:
Isn't that going to be problematic if we keep the same URI for both the WinUI 2 & WinUI 3 Gallery?

Reply:
It means we can share test infrastructure.  They weren't different before, so we wouldn't be breaking anything. We're keeping the status quo.